### PR TITLE
Prison break virus update

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

The prison break virus no longer directly messages the AI, decreasing the chance the event does nothing as the AI depowers apcs right away.
Engineering is notified about 1 minute in advance, rather than 3 minutes.
The virus now opens blast doors and shutters, so pressing the 1 brig door button in the brig no longer caps the virus in the knees. One could press this button after the doors open, and it will function normally.
Adds more variations to the virus name so it is not just 2 names.

The virus will not open every blast door due to areas, so if you have the brig locked down when it triggered, you will likely need engineering help to get the blast doors back in place.

## Why It's Good For The Game

This event is (generally) a joke. With a 3 minute warning blasted directly to the AI, the most common outcome is the AI depowers apcs and nothing happens. Or, the shutters near church are closed, so they have to run through the open brig doors where security will be. 

This changes the event so there is a chance people notice ahead of time, but generally security will need to go after the prisoners. Who likely have tracking implants and no way to open doors, so they will not be getting far.

## Testing

Ran the event. Blast doors / shutters opened, engineering still notified.

## Changelog
:cl:
tweak: Prison break no longer notifies AI 3 minutes before it begins.
tweak: Prison break can now open blast doors and shutters.
/:cl:

### CODE REVIEW

File: code/modules/events/prison_break.dm [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
 /datum/event/prison_break/announce(false_alarm)
 	if(length(areas) || false_alarm)
-		GLOB.minor_announcement.Announce("[pick("Gr3y.T1d3 virus","Malignant trojan")] detected in [station_name()] [(eventDept == "Security")? "imprisonment":"containment"] subroutines. Secure any compromised areas immediately. Station AI involvement is recommended.", "[eventDept] Alert")
+		GLOB.minor_announcement.Announce("[pick("Gr3y.T1d3 virus", "S.E.L.F program", "Malignant trojan", "Runtime error", "Cybersun worm", "[pick("Castle", "Felix", "Fractal", "Paradox", "Rubiks", "Portcullis", "Hammer", "Lockpick", "Faust")][pick(" 2.0","")] Daemon")] detected in [station_name()] [(eventDept == "Security")? "imprisonment":"containment"] subroutines. Secure any compromised areas immediately. Station AI involvement is recommended.", "[eventDept] Alert")
```
Comment (Contrabang) [inline code/modules/events/prison_break.dm:40]: :)?
```suggestion
		GLOB.minor_announcement.Announce("[pick("Gr3y.T1d3 virus", "S.E.L.F program", "Malignant trojan", "Runtime error", "Cybersun worm", "[pick("Castle", "Felix", "Fractal", "Paradox", "Rubiks", "Portcullis", "Hammer", "Lockpick", "Faust", "Dream")][pick(" 2.0","")] Daemon")] detected in [station_name()] [(eventDept == "Security")? "imprisonment":"containment"] subroutines. Secure any compromised areas immediately. Station AI involvement is recommended.", "[eventDept] Alert")
```

Comment (Qwertytoforty) [reply code/modules/events/prison_break.dm:40]: LMAO I didn't even  think of that, took me a moment!

**Comment by ATP-Engineer** [comment]
station announcement for the event proper needs to play a prison alarm seeing as how it will actually result in a prison break 90% of the time it triggers
https://www.youtube.com/watch?v=GAb7kDR0zAE

**Comment by tipoima** [comment]
> "[pick("Castle", "Felix", "Fractal", "Paradox", "Rubiks", "Portcullis", "Hammer", "Lockpick", "Faust", "Dream")][pick(" 2.0","")] Daemon")]

invisible_inc_mentioned.jpeg

**Review by Contrabang — APPROVED** [review]

File: code/game/area/areas.dm [hunk 2]
```diff
@@ -530,6 +530,9 @@
 		INVOKE_ASYNC(temp_airlock, TYPE_PROC_REF(/obj/machinery/door/airlock, prison_open))
 	for(var/obj/machinery/door/window/temp_windoor in src)
 		INVOKE_ASYNC(temp_windoor, TYPE_PROC_REF(/obj/machinery/door, open))
+	for(var/obj/machinery/door/poddoor/temp_poddoor in src)
+		INVOKE_ASYNC(temp_poddoor, TYPE_PROC_REF(/obj/machinery/door, open))
+
```
Comment (DGamerL) [inline code/game/area/areas.dm:535]:
```suggestion
```

**Review by DGamerL — APPROVED** [review]

**Pull request merged**